docs/resources/configuration_variable.md

```diff
@@ -34,10 +35,11 @@ resource "env0_configuration_variable" "drop_down" {
 ### Required
 
 - **name** (String) name to give the configuration variable
-- **value** (String) value for the configuration variable
+- **value** (String, Sensitive) value for the configuration variable
```

Comment:
What does it mean? a boolean variable? do you think it's clear enough?

Reply:
this is the value of the variable and it is sensitive
we just didn't update the docs apparently before